app/models/cart.rb

```diff
@@ -64,11 +64,11 @@ def currently_awaiting_approvers
   end
 
   def approved_approvals
-    self.approver_approvals.where(status: 'approved')
+    self.approver_approvals.approved
   end
 
   def all_approvals_received?
-    self.approver_approvals.where('status != ?', 'approved').empty?
+    self.approver_approvals.pending.empty?
```

Comment:
Hmmmmm, good catch. Maybe this method needs to be renamed, because I'm seeing it used as both "all approvals got a response", as well as "everything was approved".